### PR TITLE
Test cases for Travis CI

### PR DESCRIPTION
**Pull request by holenet**
### Be able to test django app easily.

At the root of **_Django app_**,
```
python3 manage.py test ttrs
```

Or, at the root of **_Git project_**,
```
python3 backend/manage.py test ttrs
```

You can view the coverage of the test by this command below. (at the root of **_Django app_**)
```
coverage run --source='.' manage.py test ttrs
```
And, run this command.
```
coverage report
```
You could see the coverage result like this,
image

## Travis CI
Also, I attached this repository to the Travis CI.
### So, each pull request must be able to pass the test without any errors.
I **strongly recommend** to test your implementations before sending a pull request.

### CODE REVIEW

**Comment by holenet** [comment]
I'll make coverage test

**Comment by holenet** [comment]
I made the travis CI show coverage test result.

File: backend/ttrs/urls.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
     path('bookmarked-time-tables/<int:pk>/', views.BookmarkedTimeTableDetail.as_view()),
     path('received-time-tables/', views.ReceivedTimeTableList.as_view()),
     path('received-time-tables/<int:pk>/', views.ReceivedTimeTableDetail.as_view()),
-    path('received-time-tables/<int:pk>/receive', views.ReceiveTimeTable.as_view()),
+    path('received-time-tables/<int:pk>/receive/', views.ReceiveTimeTable.as_view()),
```
Comment (Tommybro) [inline backend/ttrs/urls.py:28]: Good

**Review by Tommybro — APPROVED** [review]
Good job

**Pull request merged**